dvc/info.py

```diff
@@ -111,6 +125,26 @@ def _get_supported_remotes():
     return ", ".join(supported_remotes)
 
 
+def _get_configured_remotes(repo):
+    from urllib.parse import urlparse
+
+    from dvc.scheme import Schemes
+
+    # Get supported schemes, exclude the private variables
+    available_remotes = {
```

Comment:
Would it make sense to move this block to `scheme.py`? We are exposing internals of another class here.

Reply:
Hm, actually we don't really need use `available_remotes`.

Seems like the below code with urlparse doesn't really take into account local remotes. And also doesn't take into account `remote://` notation (this is a tribal knowledge, so no wonder it is not obvious). The most reliable way to overcome all of that is to use `get_cloud_tree`(dvc/tree/__init__.py) to resolve it for us. The only issue with it is that it tries to initialize the remote, which is not really desired, so it would be great if we cold use only the `_get_conf` part of it.

Reply:
thanks, introduced a new utility to enumerate cloud schemes for a repo in the latest commit. `info.py` looks much cleaner with this :)